docs/src/main/sphinx/release/release-473.md

```diff
@@ -0,0 +1,54 @@
+# Release 473 (19 Mar 2025)
+
+## General
+
+* Add support for array literals. ({issue}`25301`)
+* Reduce the amount of memory required for `DISTINCT` and `GROUP BY` operations. ({issue}`25127`)
+* Fix failure when setting comments on columns with upper case letters. ({issue}`25297`)
+* Fix potential query failure when `retry_policy` set to `TASK` ({issue}`25217`)
+
+## Security
+
+* Add LDAP based group provider. ({issue}`23900`)
```

Comment:
maybe 
```suggestion
* Add LDAP-based group provider. ({issue}`23900`)
```

Also .. ideally does this have docs.. then we should link to that page. Otherwise we should file an issue to backfill docs

Reply:
We don't have docs for that, yet.